documentation/source/NuRadioReco/pages/times.rst

```diff
@@ -1,31 +1,48 @@
+are then stored as floats relative to this global time inside each object. They can be obtained by the
+`get_times() <NuRadioReco.framework.base_trace.BaseTrace.get_times>` method for trace-like objects (`ElectricField <NuRadioReco.framework.electric_field.ElectricField>` ,
+`Channel <NuRadioReco.framework.channel.Channel>`), or by the `get_trigger_time <NuRadioReco.framework.trigger.Trigger.get_trigger_time>`
+method for `Trigger <NuRadioReco.framework.trigger.Trigger>` objects.
+
+Time delays are introduced by several hardware components. These time delays are often accounted for by folding/unfolding the complex transfer function (for an amplifier e.g. the measurement of the S12 parameter).
 The unfolding is typically done in the frequency domain where a convolution becomes a simple multiplication.
-As a consequence of typically short trace length (<~1000 samples) and because a Fourier transform implies implicitly a periodic signal, a pulse being at the beginning of the trace can end up being at the end of the trace.
-To avoid this behavior we often use the following procedure, through the `NuRadioReco.modules.channelStopFilter` module:
-
-We smoothly filter the first 5% and last 5% of the trace using a Tukey window function. This is a function that goes smoothly from 0 to 1.
-To avoid rollover of the pulse, we add 128ns of zeros to the beginning and end of the trace.
+As a consequence of typically short trace length (<~1000 samples) and because a Fourier transform implies implicitly a periodic signal,
+a pulse being at the beginning of the trace can end up being at the end of the trace.
+This can be avoided by using the `NuRadioReco.modules.channelStopFilter` module, which appends zeros at either end of the trace
+and applies a Tukey window to taper the ends of the trace towards zero.
 
 .. Note::
   For the **ARIANNA** experiment, the hardware produces an artifact (a glitch) at the STOP position (i.e. the physical beginning of the trace).
   Because of the way the hardware works, the STOP position is not at the beginning of the trace but can be anywhere.
   During read in of the snowshovel calibrated data files, the trace is rolled such that the physical beginning (the STOP position) is at sample zero of the trace.
-  This glitch is removed by the `channelStopFilter <NuRadioReco.modules.channelStopFilter>` procedure described above.
-
+  This glitch is removed by the `channelStopFilter <NuRadioReco.modules.channelStopFilter>` procedure described in the :ref:`module overview below <NuRadioReco/pages/times:Overview of modules that affect time>` .
+
+Station time (Event time)
+-------------------------
+The global time at which the event takes place is stored as the `event time <NuRadioReco.framework.event.Event.get_event_time>`
+in the `Event <NuRadioReco.framework.event.Event>` object.
+This time usually corrsponds to the "vertex time" of the first interaction for simulations,
```

Comment:
For data the event and station time are typically the same, right? The only exception I can think of is that we - as part of an event reconstruction - merge events across stations. Can we add a statement here - just to be very explicit.